abell-a11y-plugin/plugin/index.js

```diff
@@ -57,11 +57,23 @@ function beforeHTMLWrite(htmlText, programInfo) {
       background-color: var(--abell-a11y-bg);
       color: var(--abell-a11y-color);
     }
+
+    /*
+      Snippet to check if lists don't have any tags other than <li>
+    */
+    ul *:not(li) {
+      content: 'Unordered List can only have <li> tags';
```

Comment:
You will have to add `:before` in the selector otherwise content won't be visible.

This is how it will highlight right now:
![A border around Hey word which is inside list item](https://user-images.githubusercontent.com/30949385/91632103-24674700-e9fc-11ea-925a-c97969c2fdf5.png)

Reply:
Do we want an explicit text or just border will do ?

Reply:
if possible, text will help I believe since they will have some context of what is wrong. Also, this issue is not commonly known so can be hard to figure out what is wrong with just borders.

Reply:
Fixed : https://github.com/abelljs/official-plugins/pull/1/commits/e654c50b654094f4437a2a4cf8d460723f54228b